tardis/simulation/base.py

```diff
@@ -383,6 +387,55 @@ def run(self):
         )
         self._call_back()
 
+        self.interactive_widget()
+
+    def interactive_widget(self):
+        """
+        Generates the Interactive Widget for displaying the 
+        Plasma Data that has been generated by the simulation for 
+        each iteration
+        """
+
+        # Creating the List for no of iterations
+        iteration_number = []
+        for iteration in range(self.iterations):
```

Comment:
Use numpy arange to generate a list of integers

Reply:
This is cool. Forgot about this 🚀